### DISCUSSION TITLE
Error Message: Device with this Site, Tenant and Name exits

### DISCUSSION THREAD

**Discussion by maxzapp** [netbox-community/netbox#7340] [Q&A] [answered]
Hi, 
i got this error while documenting our lab. We have several patchpanels  (devices) with the same name but of course in different locations (rooms) and racks. Are device names really limited based on the Site (i have only one, our town) and the tenant (all patchpanels belonging of course to the same client, us)?

**Comment by candlerb** [comment, marked as answer]
[Yes](https://github.com/netbox-community/netbox/blob/v3.0.3/netbox/dcim/models/devices.py#L589).
